geometry_msgs/Pose checkPose

---
float32[] jointValues
bool valid